src/controllers/krate/search.rs

```diff
@@ -5,10 +5,11 @@ use axum::extract::FromRequestParts;
 use axum_extra::extract::Query;
 use axum_extra::json;
 use axum_extra::response::ErasedJson;
+use configuration::TsConfigurationByName;
```

Comment:
where is `configuration` coming from?

Reply:
Good catch! My editor auto-imports that, and it actually comes from `diesel_full_text_search` I've updated it to be more explicit.